### PR TITLE
[enhancement](export) filter empty partition before export table to remote storage

### PR DESCRIPTION
**Pull request by caiconghui**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by caiconghui** [comment]
run buildall

**Review by GoGoWen — DISMISSED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/load/ExportJob.java [hunk 1]
```diff
@@ -511,15 +511,23 @@ private List<List<List<Long>>> splitTablets() throws UserException {
             // get partitions
             // user specifies partitions, already checked in ExportCommand
             if (!this.partitionNames.isEmpty()) {
-                this.partitionNames.forEach(partitionName -> partitions.add(table.getPartition(partitionName)));
+                this.partitionNames.forEach(partitionName -> {
+                    Partition partition = table.getPartition(partitionName);
+                    if (partition.hasData()) {
+                        partitions.add(table.getPartition(partitionName));
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/load/ExportJob.java:517]:
```suggestion
                        partitions.add(partition);
```

**Comment by caiconghui** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by GoGoWen — APPROVED** [review]
LGTM

**Comment by caiconghui** [comment]
run p0

**Pull request merged**